ic-helper.ps1

```diff
@@ -72,17 +72,15 @@ function Find-GitRepositories {
 $foundRepository = $false
 
 try {
-    if (-not (Test-Path $path)) {
-        throw "The specified path '$path' does not exist."
-    }
-    $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+    $subdirs = Get-ChildItem -Path $path -Directory -Recurse -ErrorAction Stop
```

Comment:
Do we really need that `ErrorAction` flag here? I'm just curious, I don't know the right answer))

Reply:
From what I was seeing learning about error handling in powershell the ErrorAction flag instructs the powershell to treat any error as terminating error so, from what I understand it will stop the execution of the command after the error occurs. Now that you pointed that out, it is not necessary because the exit 1 does the same thing. My bad, I was reading to much and understood to little

Reply:
I have made more research on the ErrorAction flag and in this case it is necessary, because it will make the Get-ChildItem command to stop and not throw it's errors and display only the message in the catch block.